core/server/master/src/main/java/alluxio/master/file/activesync/ActiveSyncManager.java

```diff
@@ -0,0 +1,126 @@
+package alluxio.master.file.activesync;
+
+import alluxio.AlluxioURI;
+import alluxio.Configuration;
+import alluxio.PropertyKey;
+import alluxio.exception.InvalidPathException;
+import alluxio.heartbeat.HeartbeatContext;
+import alluxio.heartbeat.HeartbeatThread;
+import alluxio.master.file.ActiveSyncer;
+import alluxio.master.file.FileSystemMaster;
+import alluxio.master.file.meta.MountTable;
+import alluxio.resource.CloseableResource;
+import alluxio.underfs.UnderFileSystem;
+import alluxio.util.UnderFileSystemUtils;
+import alluxio.util.io.PathUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.ConcurrentHashMap;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Future;
+import java.util.stream.Collectors;
+
+public class ActiveSyncManager {
+  private static final Logger LOG = LoggerFactory.getLogger(ActiveSyncManager.class);
+  private final MountTable mMountTable;
+
+  private final List<AlluxioURI> mSyncPathList;
```

Comment:
how do these uris relate to the other maps? Can you comment on what this list is?